src/main/java/frc/trigon/robot/poseestimation/poseestimator/StandardDeviations.java

```diff
@@ -0,0 +1,58 @@
+package frc.trigon.robot.poseestimation.poseestimator;
+
+import edu.wpi.first.math.geometry.Transform2d;
+
+/**
+ * Standard Deviations represent how ambiguous an estimated pose is using calibrated gains.
+ * The higher the value is, the more ambiguous the pose is and the less trustworthy the result.
+ * Standard Deviations are used to reduce noise in a pose estimate result by accounting for how much each result is wrong by.
+ */
+public class StandardDeviations {
+    private final double translation, theta;
+
+    /**
+     * Constructs an object that stores how ambiguous the estimated pose of the robot is.
+     * The greater the number, the less trustworthy the pose is.
+     *
+     * @param translation the ambiguity of the translation aspect of the pose estimation
+     * @param theta       the ambiguity of the rotation aspect of the pose estimation
+     */
+    public StandardDeviations(double translation, double theta) {
+        this.translation = translation;
+        this.theta = theta;
+    }
+
+    /**
+     * Combines this with another {@link StandardDeviations}.
+     * This might be used when estimating a pose in relation to another estimated pose.
+     * In a case like this, you would want to find the combined standard deviations of both estimated poses to find the final standard deviations.
+     *
+     * @param other the {@link StandardDeviations} to combine with
+     * @return the combined {@link StandardDeviations}
+     */
+    StandardDeviations combineWith(StandardDeviations other) {
```

Comment:
Other?

Reply:
It's the standard for combining two of the same type of object

Reply:
Still not a good name